apps/taskman/service.py

```diff
@@ -210,6 +213,9 @@ def _generate_task_data(self, flaw: Flaw):
         else:
             summary = f"{flaw.cve_id} {flaw.title}"
 
+        if len(summary) > JIRA_SUMMARY_MAX_LENGTH:
+            summary = f"{summary[:252]}..."
```

Comment:
I do not want to be nitpicky but 252 still sounds a bit magic - it is not clear how does it relate to the rest of the statement and by just looking here one cannot guess that max length is actually 255. What about getting it by subtracting the triple dot length from the constant?